planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -702,6 +705,80 @@ PathWithLaneId BehaviorPathPlannerNode::modifyPathForSmoothGoalConnection(
   return refined_path;
 }
 
+void BehaviorPathPlannerNode::visualizeDrivableAreasWithSharedLanelets()
```

Comment:
@zulfaqar-azmi-t4 It'd be better to move this function into util file. So that just the process flow is written in the `behavior_path_planner_node` file. What do you think?

Reply:
I was thinking of moving the function to `route_handler` since it is more related to routing. I'll move them and re-update the branch.